tools/packaging/kata-deploy/local-build/dockerbuild/Dockerfile

```diff
@@ -40,3 +43,17 @@ RUN apt-get update && \
 
 ENV USER ${IMG_USER}
 USER ${UID}:${GID}
+
+RUN mkdir ~/.docker/
+RUN sh -c "echo ' \
```

Comment:
Can this be rewritten like this?

```bash
sh -c 'cat << EOF > ~/.docker/config.json
{
  "proxies": {
    "default": {
      "httpProxy": "${http_proxy}",
      "httpsProxy": "${https_proxy}",
      "noProxy": "localhost,127.0.0.1"
    }
  }
}
EOF'
```

Too many back quotas are not readable and easy to run into errors.

Reply:
@liubin Thanks, I will check it again.